PixiEditor/Helpers/SupportedFilesHelper.cs

```diff
@@ -0,0 +1,101 @@
+﻿using PixiEditor.Models;
+using System;
+using System.Collections.Generic;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+
+namespace PixiEditor.Helpers
+{
+    public class SupportedFilesHelper
+    {
+        static ImageFormat[] _imageFormats = new[] { ImageFormat.Png, ImageFormat.Jpeg, ImageFormat.Bmp, ImageFormat.Gif, /*ImageFormat.Tiff */};
+        static Dictionary<string, List<string>> extensions;
+        public static ImageFormat[] ImageFormats { get => _imageFormats; }
+
+        static SupportedFilesHelper()
+        {
+            extensions = new Dictionary<string, List<string>>();
+            extensions[Constants.NativeExtension] = new List<string>() { Constants.NativeExtension };
+            foreach(var format in _imageFormats)
+                extensions[Format2Extension(format)] = GetFormatExtensions(format);
+        }
+
+        public static IEnumerable<string> GetAllSupportedExtensions()
+        {
+            return extensions.SelectMany(i => i.Value);
+        }
+
+        public static List<string> GetExtensions()
+        {
+            return extensions.Keys.ToList();
+        }
+
+        public static List<string> GetFormatExtensions(ImageFormat format)
+        {
+            var res = new List<string>();
+            res.Add(Format2Extension(format));
+            if (format == ImageFormat.Jpeg)
+                res.Add(".jpg");
+            return res;
+        }
+
+        public static string Format2Extension(ImageFormat format)
```

Comment:
Please don't use a 2 to replace the word `To`

Reply:
Maybe we shall we add "Coding Standards" to CONTRIBUTING.md file, so that new man can start smoothly ?